src/pybind/mgr/dashboard/controllers/home.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import os
+import json
+
+import cherrypy
+from cherrypy.lib.static import serve_file
+
+from . import Controller, UiApiController, BaseController, Proxy, Endpoint
+from .. import mgr, logger
+
+
+LANGUAGES = {f for f in os.listdir(mgr.get_frontend_path()) \
+             if os.path.isdir("{}/{}".format(mgr.get_frontend_path(), f))}
+
+
+def _get_default_language():
+    with open("{}/../package.json".format(mgr.get_frontend_path()), "r") as f:
+        config = json.load(f)
```

Comment:
The idea is great, but are we sure we have `package.json` in dist packages (I though only `dist/` contents were packaged)?

Reply:
Currently the whole frontend directory is being packaged (unnecessarily IMO). If in the future we fix the frontend packaging, what would be the best way to get the default language value?

Reply:
Mmm, if we want to keep front-end & back-end decoupled, we'd need a separate config file setting for Python.